hikari/events/guild_events.py

```diff
@@ -265,6 +265,14 @@ class GuildUpdateEvent(GuildEvent):
     shard: gateway_shard.GatewayShard = attr.ib(metadata={attr_extensions.SKIP_DEEP_COPY: True})
     # <<inherited docstring from ShardEvent>>.
 
+    old_guild: typing.Optional[guilds.Guild] = attr.ib()
+    """Old guild object from cache or 'builtins.None'.
+
+    Returns
+    -------
+    hikari.guilds.Guild
```

Comment:
would we not usually have descriptions for the returned field even if it just essentially dupes the top bit?